ibeaconscanner/src/main/AndroidManifest.xml

```diff
@@ -3,12 +3,27 @@
     xmlns:android="http://schemas.android.com/apk/res/android">
 
     <uses-permission android:name="android.permission.BLUETOOTH"/>
+    <uses-permission android:name="android.permission.WAKE_LOCK"/>
     <uses-permission android:name="android.permission.BLUETOOTH_ADMIN"/>
-    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
 
     <application
         android:label="@string/app_name">
 
+        <receiver
+            android:name=".BluetoothScanBroadcastReceiver"
+            android:exported="false">
+            <intent-filter>
+                <action android:name="android.intent.action.BOOT_COMPLETED"/>
+                <action android:name="android.intent.action.ACTION_POWER_CONNECTED"/>
+                <action android:name="android.intent.action.ACTION_POWER_DISCONNECTED"/>
+            </intent-filter>
+        </receiver>
+
+        <service
+            android:exported="false"
+            android:name=".OnExitJobService"/>
```

Comment:
did you check the exported package name? (so that 2 apps can have this library)

Reply:
should be no problem since it is an unexported service, so it is containted in the standbox of the app